Assets/HoloToolkit/Input/Scripts/GameControllerManipulator.cs

```diff
@@ -0,0 +1,160 @@
+﻿using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Allows the user to place and rotate GameObjects using a game controller.
+    /// </summary>
+    /// <remarks>Make sure to enable the HumanInterfaceDevice capability before using.</remarks>
+    public class GameControllerManipulator : MonoBehaviour
+    {
+        [Tooltip("Name of the joystick axis that moves the object along X as set in InputManager")]
+        public string MoveXAxisName = "ControllerLeftStickX";
+        [Tooltip("Speed of movement along X")]
```

Comment:
Could we add some single spaces between our fields and the tooltip of the next field please?